python/dgl/heterograph.py

```diff
@@ -3262,10 +3262,11 @@ def edge_ids(self, u, v, return_uv=False, etype=None):
                 # Raise error since some (u, v) pair is not a valid edge.
                 idx = F.nonzero_1d(is_neg_one)
                 raise DGLError(
-                    "Error: (%d, %d) does not form a valid edge."
+                    "Error: %d invalid edges, (%d, %d) is one."
                     % (
-                        F.as_scalar(F.gather_row(u, idx)),
-                        F.as_scalar(F.gather_row(v, idx)),
+                        len(idx),
+                        F.as_scalar(F.gather_row(u, idx[0])),
```

Comment:
I think it's ok to print all the invalid node pairs.

Reply:
It can't now, there will be an error "ValueError: only one element tensors can be converted to Python scalars" when calling as_scalar(), and will not print any data, so I change it.